src/main/java/frc/robot/notification/ChangeColor.java

```diff
@@ -0,0 +1,27 @@
+package frc.robot.notification;
+
+import edu.wpi.first.wpilibj2.command.Command;
+
+public class ChangeColor extends Command {
+  private LEDSubsystem subystem;
+  
+  public ChangeColor(){
+    subystem = new LEDSubsystem();
+  }
+
+  @Override
+  public void initialize(){
+    subystem.handleStateUpdate(LEDStateEnum.COLOR_GREEN_SPARK);
+  }
+
+  @Override
+  public void execute(){
+
+  }
+
+  @Override
+  public void end(boolean interrupeted){
+
+  }
```

Comment:
Should explicitly declare `isFinished()` (return true). Could delete end and execute since they do nothing